common/api/imodeljs-backend.api.md

```diff
@@ -2887,6 +2888,15 @@ export class LocalhostIpcHost {
     }): Promise<void>;
 }
 
+// @beta
+export class LockConflict extends IModelError {
+    constructor(
+    briefcaseId: BriefcaseId,
+    briefcaseAlias: string, msg: "shared lock is held" | "exclusive lock is already held");

```

Comment:
Should this use `message` property of `BentleyError `class? I do not see `msg` property in the inheritance chain.

Reply:
it does. It is a bit confusing to look at these .md files. They're generated for the api checker and only show the signatures/properties.